tests/ValueObjectTest.php

```diff
@@ -265,6 +309,10 @@ public function paymentMethodCodeProvider()
         );
     }
 
+    /**
+     * @param int $length
```

Comment:
It is not maybe important, but the length should be greater than 0 and the result should be non-empty-string, otherwise it makes no sense to generate empty string.

Reply:
Good idea, I add only not breaking change annotations, but this is test and here is break changes fine.